### DISCUSSION TITLE
Make a module for 4500

### DISCUSSION THREAD

**Discussion by KoRn0509** [PycmShoma/IncreasedBatteryCapacity#4] [Q&A]
Make a module for 4500 pls, AOSP and MIUI 
or maybe you can tell me how to do it yourself?

**Comment by PycmShoma** [comment]
I'll take a look next week

**Comment by PycmShoma** [comment]
Done. 
Added 4500 mAh versions
https://github.com/PycmShoma/IncreasedBatteryCapacity/releases

**Comment by Arnabs2008** [comment]
I have upgraded my Poco F3 (alioth) battery to a 5000mAh, but i guess the battery is charging only to the default 4500mAh. How to make a similar module for my device ?
